indigo_api/migrations/0129_migrate_refs_uris.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-04-22 12:39
+from __future__ import unicode_literals
+
+from lxml import etree
+
+from django.db import migrations
+
+from cobalt import Act, FrbrUri, datestring
+from cobalt.uri import FRBR_URI_RE
+from indigo.analysis.refs.base import BaseRefsFinder
+
+
+def new_frbr_uri(uri, forward):
+    """ Generates a cobalt FrbrUri object using all the existing URI's attributes.
+        If `forward` is True, the optional /akn prefix is included; otherwise it isn't.
+    """
+    if not isinstance(uri, FrbrUri):
+        uri = FrbrUri.parse(uri)
+    prefix = None
+    if forward:
+        prefix = 'akn'
+
+    return FrbrUri(
+        prefix=prefix,
+        country=uri.country,
+        locality=uri.locality,
+        doctype=uri.doctype,
+        subtype=uri.subtype,
+        actor=uri.actor,
+        date=uri.date,
+        number=uri.number,
+        work_component=uri.work_component,
+        language=uri.language,
+        expression_date=uri.expression_date,
+        expression_component=uri.expression_component,
+        expression_subcomponent=uri.expression_subcomponent,
+        format=uri.format
+    )
+
+
+def set_frbr_uri_meta_ident(doc, uri):
```

Comment:
Why not just load the document as a Cobalt doc, and then set the new FRBR URI, and let cobalt do all this work?

Reply:
sounds great! how do I set the new FRBR URI?

Reply:
got it!

Reply:
(I assumed it'd be more complicated than just literally setting it)